apps/explorer/src/ui/modal/Modal.tsx

```diff
@@ -5,15 +5,47 @@ import { Dialog, Transition } from '@headlessui/react';
 import { X12 } from '@mysten/icons';
 import { Fragment, type ReactNode } from 'react';
 
-import { IconButton } from './IconButton';
+import { Heading } from '../Heading';
 
-export interface LightBoxProps {
+export interface ModalProps {
     open: boolean;
     onClose: () => void;
     children: ReactNode;
 }
 
-export function LightBox({ open, onClose, children }: LightBoxProps) {
+export function CloseButton({ onClick }: { onClick: () => void }) {
+    return (
+        <button
+            onClick={onClick}
+            type="button"
+            className="absolute right-0 top-0 p-4 text-steel hover:text-steel-darker"
+        >
+            <X12 />
+        </button>
+    );
+}
+
+export function ModalBody({ children }: { children: ReactNode }) {
+    return <div className="py-5">{children}</div>;
+}
+
+export function ModalContent({ children }: { children: ReactNode }) {
+    return (
+        <div className="flex flex-col rounded-lg bg-gray-40 p-5">
+            {children}
+        </div>
+    );
+}
+
+export function ModalHeading({ children }: { children: ReactNode }) {
```

Comment:
Seems like these other modal components aren't used? Not sure if that's intentional